scripts/glue/load-s3-to-db.py

```diff
@@ -0,0 +1,114 @@
+import os
+import sys
+import boto3
+import logging
+import psycopg2 
+import pandas as pd
+from io import BytesIO, StringIO
+from argparse import ArgumentParser
+from datetime import datetime, timedelta
+
+# Initialize logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
+def load_environment_variables():
+    try:
+        # if this is run in glue, use the AWS Glue provided env variables
+        # for reference: https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-crawler-pyspark-extensions-get-resolved-options.html
+        from awsglue.utils import getResolvedOptions
+        args = getResolvedOptions(sys.argv, ['DB_HOST', 'DB_PORT', 'POSTGRES_DB', 'POSTGRES_USER', 'POSTGRES_PASSWORD'])
+        return args
+    except Exception:
+        # if run locally, use the .env file 
+        from dotenv import load_dotenv
+        load_dotenv()  # Fallback to .env file if not running on AWS Glue
+        return {
+            'DB_HOST': os.getenv('DB_HOST'),
+            'DB_PORT': os.getenv('DB_PORT'),
+            'POSTGRES_DB': os.getenv('POSTGRES_DB'),
+            'POSTGRES_USER': os.getenv('POSTGRES_USER'),
+            'POSTGRES_PASSWORD': os.getenv('POSTGRES_PASSWORD')
+        }
+
+def is_s3_path(file_path):
+    return file_path.startswith('s3://')
+
+def read_csv(file_path):
+    logging.info("Reading data from %s", file_path)
+    try:
+        if is_s3_path(file_path):
+            s3 = boto3.client('s3')
+            bucket_name = file_path.split('/')[2]
+            s3_path = '/'.join(file_path.split('/')[3:])
+            obj = s3.get_object(Bucket=bucket_name, Key=s3_path)
+            df = pd.read_csv(BytesIO(obj['Body'].read()))
+        else:
+            df = pd.read_csv(file_path)
+    except Exception as e:
+        logging.error("Error reading CSV file: %s", e)
+        sys.exit(1)
+
+    # Remove the 'Unnamed: 0' column if it exists
+    if 'Unnamed: 0' in df.columns:
+        df = df.drop(columns=['Unnamed: 0'])
+
+    return df
+
+def process_data(df):
```

Comment:
There is no need for excessive comments here.

Your code should be self-documented.

So here you naming stuff like `df`, `pd` - I actually have no clue what the code is trying to do actually

Please watch the following video: [link](https://www.youtube.com/watch?v=Bf7vDBBOBUA)